HW2_Grigoriants/calculator.py

```diff
@@ -0,0 +1,32 @@
+### This will be our great calculator. Please contribute by writing one of the functions
+def division(num1, num2): #division of numbers
+    if num2 == 0:
+        return "Dude, for real?"
```

Comment:
Когда вы проводите операции над числами, вы ожидаете, что на выходе получите другое число. Здесь же вернется строка. В таких ситуациях лучше останавливать программу и выбрасывать ошибку. Например:
```suggestion
print('Dude, for real?', file=sys.stderr)
os.exit(1)
```

С модулями sys и os вы еще познакомитесь:)